cafebabel/templates/articles/tags/detail.html

```diff
@@ -17,15 +17,15 @@
   {{ meta('property=og:title', self.meta_title()) }}
   {{ meta('property=og:type', 'article') }}
   {% if tag.summary %}
-    {{ meta('property=og:description', tag.summary) }}
+    {{ meta('property=og:description', tag.summary|striptags) }}
```

Comment:
Wondering what the description should be when no summary exist?

Reply:
nothing?

Reply:
To me there is the test and that's OK not to have a description.